### PR TITLE
Feature/36 community event

### PR DESCRIPTION
**Pull request by Pete-Robelou**
- Event layout
- Updated READMEs for site, blog posts, case studies, evenst and press releases
- Agenda CSS component
- Utility CSS updates

### CODE REVIEW

File: src/en-GB/community/events/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+```
+
+- `title` (String) — Determines the link text as it appears in the event navigation card. Also used as the `<title>` for the page.\*
+- `date` (Date) — Determines the start date of the event.
+- `end` (Date) — Determines the end date of the event.
+- `location` (String) — Determines the location of the event, city and country.
+- `venue` (String) — Determines the venue of the event, building name and street.
+- `map` (String) — Determines the map of the event with a google maps link.\*
+- `register` (String) — Determines the website to register for the event. This is hidden once the end date of the event has pasted.
```
Comment (adamduncan) [inline src/en-GB/community/events/README.md:47]:
```suggestion
- `register` (String) — Determines the website to register for the event. This is hidden once the end date of the event has passed.
```

File: src/en-GB/community/events/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+```
+
+- `title` (String) — Determines the link text as it appears in the event navigation card. Also used as the `<title>` for the page.\*
+- `date` (Date) — Determines the start date of the event.
+- `end` (Date) — Determines the end date of the event.
+- `location` (String) — Determines the location of the event, city and country.
+- `venue` (String) — Determines the venue of the event, building name and street.
+- `map` (String) — Determines the map of the event with a google maps link.\*
```
Comment (adamduncan) [inline src/en-GB/community/events/README.md:46]:
```suggestion
- `map` (String) — Google Maps URL for the event venue location\*
```

Just a convention thing, but wonder if these could be more declaritive/explicit. Might save on repeating "Determines"

Side note: Do you think we could derive the Google Maps link from the Venue + Location information? I.e. `https://www.google.com/maps/search/ibis+Styles+Wroclaw+Centrum` ([Docs](https://developers.google.com/maps/documentation/urls/get-started#search-action) re: escaping characters, etc.)

File: src/en-GB/community/events/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- `image` (String) — Determines the asset path for the image shown in the event navigation card and the hero image beneath the title on the event page.
+- `links` (List) - These only appear when the end date of the event has pasted.
+  - `videos` (String) — Determines the website showing videos of the event with a youtube/vimeo link.
+  - `slides` (String) — Determines the website or PDF showing slides of the event.
+- `sponsors` (Array)
+  - `label` (String) — Determines the type of sponsors for the event.
+    `list`
+    - `name` (String) — Determines the name of the sponsor.
+      `logo` (String) — Determines the asset path for the sponsor logo.
+      `website` (String) — Determines the URL of the sponsor website which is applied to the sponsor logo.
```
Comment (adamduncan) [inline src/en-GB/community/events/README.md:57]:
```suggestion
    - `list`
        - `name` (String) — Determines the name of the sponsor.
        - `logo` (String) — Determines the asset path for the sponsor logo.
        - `website` (String) — Determines the URL of the sponsor website which is applied to the sponsor logo.
```

Might just need to tweak the indentent list levels here to ensure the hierarchy is correct.

**Review by adamduncan — APPROVED** [review]
Looking good, @Pete-Robelou.

Like with Blogs, I wonder whether at some point we'll need to consider an `intro` property for event frontmatter, so we can the intro paragraphs in place to achieve the "Heading, Intro, links, image, richtext" order of elements as visualised in the designs. Let's see.

**Comment by Pete-Robelou** [comment]
> Like with Blogs, I wonder whether at some point we'll need to consider an `intro` property for event frontmatter, so we can the intro paragraphs in place to achieve the "Heading, Intro, links, image, richtext" order of elements as visualised in the designs. Let's see.

It's possible. It'll be an easy update but let's see how things progress first.

File: src/en-GB/community/events/README.md (hunk 3 above)
Comment (Pete-Robelou) [reply src/en-GB/community/events/README.md:57]: Looks like you maybe seeing different indentation to me. When I look at the README it appears the same as your revised indentation.

```
sponsors:
  - label: Platinum sponsors
    list:
      - name: SoftIron
        logo: "/assets/image.png"
        website: "https://softiron.com"
      - name: RedHat
        logo: "/assets/image.png"
        website: "https://redhat.com"
  - label: Gold sponsors
    list:
      - name: SUSE
        logo: "/assets/image.png"
        website: "https://suse.com"
```

File: src/en-GB/community/events/README.md (hunk 1 above)
Comment (Pete-Robelou) [reply src/en-GB/community/events/README.md:47]: Good spot. I've updated 🙂

File: src/en-GB/community/events/README.md (hunk 2 above)
Comment (Pete-Robelou) [reply src/en-GB/community/events/README.md:46]: I've revised the `determines` language across all the current READMEs.

I like the idea of inferring the maps link from the venue and location. 

@adamduncan I've implemented it like this, what do you think?

{% set map = venue + '-' + location %}

`https://www.google.com/maps/search/?api=1&query={{ map | slug }}`

**Pull request merged**